doc/nrf/ug_nrf9160.rst

```diff
@@ -24,33 +24,94 @@ For more details on the SiP, see the `nRF9160 product website`_.
 
 The figure illustrates the conceptual layout when targeting
 an nRF9160 Cortex-M33 application MCU with TrustZone.
-The M33 TrustZone divides the MCU into secure and non-secure domains.
-When the MCU boots, it always starts executing from the secure area,
-which in this case is the **Secure Partition Manager**.
-The **Secure Partition Manager** configures a part of memory and peripherals to be
-non-secure and then jumps to the main application located in the non-secure area.
-The **BSD Socket** library must be embedded in the application because
-BSD sockets are the primary interface to the LTE modem.
+
+Application MCU
+===============
+
+The M33 TrustZone divides the application MCU into secure and non-secure domains.
+When the MCU boots, it always starts executing from the secure area.
+The secure bootloader chain starts the :ref:`nrf9160_ug_secure_partition_manager`, which configures a part of memory and peripherals to be non-secure and then jumps to the main application located in the non-secure area.
+
+In Zephyr, :ref:`nrf9160_pca10090` is divided into two different boards:
+
+* ``nrf9160_pca10090`` for firmware in the secure domain
+* ``nrf9160_pca10090ns`` for firmware in the non-secure domain
+
+Make sure to select the suitable board name when building your application.
+
+
+Secure bootloader chain
+-----------------------
+
+A secure bootloader chain protects your application against running unauthorized code, and it enables you to do device firmware updates (DFU).
+See :ref:`ug_bootloader` for more information.
+
+A bootloader chain is optional.
+Not all of the nRF9160 samples include a secure bootloader chain, but the ones that do use the :ref:`bootloader` sample and :doc:`mcuboot:index`.
 
 .. _nrf9160_ug_secure_partition_manager:
 
 Secure Partition Manager
-========================
+------------------------
 
-To use any of the nRF9160 samples, you must first compile and flash the
-**Secure Partition Manager** sample.
+All nRF9160 samples require the :ref:`secure_partition_manager` sample.
 It provides a reference implementation of a Secure Partition Manager firmware.
 This firmware is required to set up the nRF9160 DK so that it can run user applications in the non-secure domain.
 
-Note that the **Secure Partition Manager** sample is automatically included in the build for the nrf9160_pca10090ns board.
-To disable the automatic inclusion of the **Secure Partition Manager** sample, set the option ``CONFIG_SPM=n`` in the project configuration.
+Note that the Secure Partition Manager sample is automatically included in the build for the ``nrf9160_pca10090ns`` board.
```

Comment:
I would get rid of `Note that`. I think it's just a filler that makes longer a sentence that is not suitable to be a `.. note` .